test/intl402/BigInt/prototype/toLocaleString/taint-Intl-NumberFormat.js

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013-2019 Mozilla Corporation; Igalia, S.L. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
```

Comment:
Got a "No license found in: intl402/BigInt/prototype/toLocaleString/taint-Intl-NumberFormat"
I think you need to change the line "// This code is governed by the license found in the LICENSE file." to
something like "// This code is governed by the BSD license found in the LICENSE file."